skimage/segmentation/_clear_border.py

```diff
@@ -39,26 +41,47 @@ def clear_border(labels, buffer_size=0, bgval=0, in_place=False):
            [0, 0, 1, 1, 1, 1, 1, 0, 0],
            [0, 1, 1, 1, 1, 1, 1, 1, 0],
            [0, 0, 0, 0, 0, 0, 0, 0, 0]])
+    >>> mask = np.array([[1, 0, 0, 0, 0, 0, 0, 0, 0],
+    ...                  [1, 0, 0, 0, 0, 0, 0, 0, 0],
+    ...                  [1, 0, 0, 0, 0, 0, 0, 0, 0],
+    ...                  [1, 0, 0, 0, 0, 0, 0, 0, 0],
+    ...                  [1, 0, 0, 0, 0, 0, 0, 0, 0],
+    ...                  [1, 1, 1, 1, 1, 1, 1, 1, 1]]).astype(np.bool)
+    >>> clear_border(labels, mask=mask)
+    array([[0, 0, 0, 0, 0, 0, 0, 1, 0],
+           [0, 0, 0, 0, 1, 0, 0, 0, 0],
+           [0, 0, 0, 1, 0, 1, 0, 0, 0],
+           [0, 0, 1, 1, 1, 1, 1, 0, 0],
+           [0, 1, 1, 1, 1, 1, 1, 1, 0],
+           [0, 0, 0, 0, 0, 0, 0, 0, 0]])
```

Comment:
I think the docstring example has a lot of room for improvement, actually. (Sorry, this isn't your fault, but it's a good opportunity to tweak it.) Specifically, I think the border objects should be extended by a pixel to show that they just need to *touch* the border/mask to be removed, rather than it just being a single-pixel-wide removal.

Reply:
Ok, I think I understood what you meant. I did a small tweak where I increased the size of the border objects. Should the example be improved still?